### ISSUE TITLE
How did you train Mask-RCNN for RGBD benchmark dataset

### ISSUE THREAD

**Issue by hoangcuongbk80** [martinruenz/maskfusion#9]
Hi there,
Thanks for your contribution.
Could you explain how did you get annotated data and train RGBD benchmark dataset?
As I know, TUM dataset only contains trajectory ground truth camera pose without labelled frames.

**Comment by martinruenz** [comment]
Hey, we used pre-trained weights for the MS COCO dataset, see: https://github.com/matterport/Mask_RCNN
